src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorDocumentManager.cs

```diff
@@ -3,27 +3,23 @@
 
 using System;
 using System.Collections.Generic;
-using System.ComponentModel.Composition;
 using System.Diagnostics;
 using Microsoft.CodeAnalysis.Razor;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
 
 namespace Microsoft.VisualStudio.Editor.Razor
 {
-    [System.Composition.Shared]
-    [Export(typeof(RazorDocumentManager))]
     internal class DefaultRazorDocumentManager : RazorDocumentManager
     {
         private readonly ForegroundDispatcher _foregroundDispatcher;
         private readonly RazorEditorFactoryService _editorFactoryService;
         private readonly TextBufferProjectService _projectService;
 
-        [ImportingConstructor]
         public DefaultRazorDocumentManager(
             RazorEditorFactoryService editorFactoryService,
             TextBufferProjectService projectService,
-            VisualStudioWorkspaceAccessor workspaceAccessor)
+            ForegroundDispatcher dispatcher)
```

Comment:
put dispatcher first for consistency across our types. You can utilize the refactor method dialog to do this so you don't have to change all references manually.

Reply:
Done